migrations/20220809015149-create-program-location.js

```diff
@@ -23,6 +37,10 @@ module.exports = {
         type: Sequelize.DATE,
       },
     });
+    await queryInterface.addIndex('ProgramLocations', {
+      fields: ['LocationId', 'ProgramId'],
+      unique: true,
+    });
```

Comment:
An INDEX is a data structure that the database uses to optimize lookups in a table- it can also be used to enforce certain CONSTRAINTS, such as a UNIQUE constraint. Here, the combination of the LocationId and ProgramId together must be unique in every row/record in the database. This prevents duplicates...